master broken

[ 57%] Building CXX object src/CMakeFiles/opalx.dir/Utilities/AbstractSpline.cpp.o
In file included from /data/user/adelmann/NightlyBuildX/workspace/master/src/src/Utilities/AbstractSpline.cpp:18:
/data/user/adelmann/NightlyBuildX/workspace/master/src/src/Utilities/AbstractSpline.h:37:9: error: 'size_t' does not name a type
   37 |         size_t last_index_{};
      |         ^~~~~~

--- a/src/Utilities/AbstractSpline.h
+++ b/src/Utilities/AbstractSpline.h
@@ -18,6 +18,7 @@
 #ifndef OPALX_ABSTRACTSPLINE_H
 #define OPALX_ABSTRACTSPLINE_H
 
+#include <vector>
 #include "OPALTypes.h"
 
 /// \brief Base class for the linear and cubic interpolation spline classes

